integration/test/sanity/sanity_test.go

```diff
@@ -1,17 +1,15 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
-//go:build linux && integration
-// +build linux,integration
+//go:build integration
+// +build integration
 
 package sanity
 
 import (
 	"testing"
-
-	"github.com/aws/amazon-cloudwatch-agent/integration/test"
 )
 
 func TestAgentStatus(t *testing.T) {
-	test.RunShellScript("resources/verifyLinuxCtlScript.sh")
+	SanityCheck()
```

Comment:
Nit - We should be using the `testing.T` struct to assert failure rather than relying on `log.Fatal`